### PR TITLE
Fix team responsive

### PR DESCRIPTION
**Pull request by Morel-Victor**

### CODE REVIEW

**Review by NacymeG — DISMISSED** [review]
Ok for me

**Review by NacymeG — DISMISSED** [review]
Ok for me

File: src/Components/Header/style.jsx [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+
+    @media (max-width: 1400px) {
+      display: flex;
+      flex-direction: column;
+      text-align: center;
+      margin-top: 0;
+      padding: 70px 0 0 0;
+      font-size: 35px;
+    }
+
```
Comment (Hacene010) [inline src/Components/Header/style.jsx]: empty line !

File: src/Components/Header/style.jsx [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+  background-color: var(--color--secondary);
+  position: fixed;
+  z-index: 10;
+  top: 0;
+  ul {
+    color: var(--color--thirdary);
+    text-decoration: none;
+    margin-top: 37px;
+    padding-right: 20px;
+
```
Comment (Hacene010) [inline src/Components/Header/style.jsx]: empty line !

File: src/Components/Header/style.jsx [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+  color: var(--color--primary);
+  min-width: 50px;
+  max-width: 50px;
+  min-height: 75px;
+  max-height: 75px;
+  margin-right: 10px;
+  :hover {
+    cursor: pointer;
+  }
+
```
Comment (Hacene010) [inline src/Components/Header/style.jsx]: empty line !

File: src/Components/Header/style.jsx [hunk 4]
```diff
@@ -0,0 +1,247 @@
...
+  @media (max-width: 768px) {
+    height: 35px;
+    margin-right: 20px;
+  }
+`;
+
+const Row = styled.div`
+  display: flex;
+  flex-direction: row;
+
```
Comment (Hacene010) [inline src/Components/Header/style.jsx]: empty line !

File: src/Components/Header/style.jsx [hunk 5]
```diff
@@ -0,0 +1,247 @@
...
+  & > * {
+    height: 75px;
+  }
+
```
Comment (Hacene010) [inline src/Components/Header/style.jsx]: empty line !

**Review by Hacene010 — CHANGES_REQUESTED** [review]
Please check your style.jsx files and put blank lines everywhere or remove them everywhere to be consistent.

**Review by NacymeG — DISMISSED** [review]

**Review by Hacene010 — DISMISSED** [review]

**Review by Hacene010 — APPROVED** [review]

**Review by NacymeG — APPROVED** [review]

**Pull request merged**